tests/devices/test_default_gaussian.py

```diff
@@ -398,79 +398,26 @@ def test_observable_map(self, gaussian_dev):
-            phi_r = -0.124
-
-            gaussian_dev.apply("DisplacedSqueezedState", wires=Wires([0]), par=[a, phi_a, r, phi_r])
-            gaussian_dev.apply("DisplacedSqueezedState", wires=Wires([1]), par=[a, phi_a, r, phi_r])
-
-            # get the equivalent pennylane operation class
-            op = qml.ops.__getattribute__(gate_name)
-            # the list of wires to apply the operation to
-            w = list(range(op.num_wires))
-
-            if op.par_domain == "A":
-                # the parameter is an array
-                if gate_name == "GaussianState":
-                    cov = np.diag([0.5234] * 4)
-                    mu = np.array([0.432, 0.123, 0.342, 0.123])
-                    p = [cov, mu]
-                    w = list(range(2))
-                    expected_out = [cov, mu]
-                elif gate_name == "InterferometerUnitary":
-                    w = list(range(2))
-                    p = [U]
-                    S = fn(*p)
-                    expected_out = S @ gaussian_dev._state[0] @ S.T, S @ gaussian_dev._state[1]
-            else:
-                # the parameter is a float
-                p = [0.432423, -0.12312, 0.324, 0.751][: op.num_params]
-
-                if gate_name == "Displacement":
-                    alpha = p[0] * np.exp(1j * p[1])
-                    state = gaussian_dev._state
-                    mu = state[1].copy()
-                    mu[w[0]] += alpha.real * np.sqrt(2 * hbar)
-                    mu[w[0] + 2] += alpha.imag * np.sqrt(2 * hbar)
-                    expected_out = state[0], mu
-                elif "State" in gate_name:
-                    cov, mu = fn(*p, hbar=hbar)
-                    expected_out = gaussian_dev._state
-                    expected_out[1][[w[0], w[0] + 2]] = mu
-
-                    ind = np.concatenate([np.array([w[0]]), np.array([w[0]]) + 2])
-                    rows = ind.reshape(-1, 1)
-                    cols = ind.reshape(1, -1)
-                    expected_out[0][rows, cols] = cov
-                else:
-                    # if the default.gaussian is an operation accepting parameters,
-                    # initialise it using the parameters generated above.
-                    S = fn(*p)
-
-                    # calculate the expected output
-                    if op.num_wires == 1:
-                        # reorder from symmetric ordering to xp-ordering
-                        S = block_diag(S, np.identity(2))[:, [0, 2, 1, 3]][[0, 2, 1, 3]]
-
-                    expected_out = S @ gaussian_dev._state[0] @ S.T, S @ gaussian_dev._state[1]
-
-            gaussian_dev.apply(gate_name, wires=Wires(w), par=p)
-
-            # verify the device is now in the expected state
-            assert gaussian_dev._state[0] == pytest.approx(expected_out[0], abs=tol)
-            assert gaussian_dev._state[1] == pytest.approx(expected_out[1], abs=tol)
+        gaussian_dev = qml.device("default.gaussian", wires=1)
+
```

Comment:
My thinking is this: if the apply function works, then the principle works. 

[TODO Maria: add three tests for Displacement, GaussianState, other State]

Reply:
yes! If this is a unit test testing the `apply` method, we only should be checking that `apply` is doing as needed. There is no need to test every supported gate through `apply`.